Cursor/funkybot_Mouse Buddy.lua

```diff
@@ -0,0 +1,805 @@
+-- @description Mouse Buddy
```

Comment:
To match the convention of other scripts and native REAPER actions:
```suggestion
-- @description Mouse buddy
```